src/components/core/list/types/index.ts

```diff
@@ -0,0 +1,60 @@
+import React, { HTMLAttributes } from 'react';
+
+export type TListType = 'numbered' | 'bulleted' | 'checklist' | 'compound';
```

Comment:
[Comment #1] Since we have some string checks based on this, can we change this to an `enum`?

Reply:
unfortunately we can't use `enum` here, since the type prop is used by the stitches component. I can use `enum` here but it will be more of a overhead TBH. because then I have to convert back the `enum` to string literals.